SUMMARY.md

```diff
@@ -1,63 +1,64 @@
-# Summary
+\*\*# Summary
 
 ## Course
 
-* Before you start
-  * [Week 0 - Introduction and basic setup](/others/lesson0.md)
-* [HTML/CSS](/html-css/index.md)
-  * [Week 1 - Semantic HTML and CSS](/html-css/week-1/lesson.md)
-  * [Week 2 - Responsive Web and layout](/html-css/week-2/lesson.md)
-  * [Week 3 - Design frameworks, Naming patterns and Forms](/html-css/week-3/lesson.md)
-* [JavaScript I](/js-core-1/index.md)
-  * [Week 4 - Hello JavaScript](/js-core-1/week-1/lesson.md)
-  * [Week 5 - Conditionals and Arrays](/js-core-1/week-2/lesson.md)
-  * [Week 6 - More Arrays](/js-core-1/week-3/lesson.md)
-* [JavaScript II](/js-core-2/index.md)
-  * [Week 7 - Objects](/js-core-2/week-1/lesson.md)
-  * [Week 8 - JS in the Browser](/js-core-2/week-2/lesson.md)
-  * [Week 9 - More JS in the Browser](/js-core-2/week-3/lesson.md)
-* [JavaScript III](/js-core-3/index.md)
-  * [Week 10 - Unit Tests and TDD](/js-core-3/week-1/lesson.md)
-  * [Week 11 - More TDD, ES6](/js-core-3/week-2/lesson.md)
-  * [Week 12 - Recap](/js-core-3/week-3/lesson.md)
-* [React](/react/index.md)
-  * [Week 13 - React 101](/react/week-1/lesson.md)
-  * [Week 14 - Reacting to Changes](/react/week-2/lesson.md)
-  * [Week 15 - Fetching Data](/react/week-3/lesson.md)
-  * [Routing (optional)](/react/week-4/lesson.md)
-* [Node](/node/index.md)
-  * [Week 16 - Node and Express 101](/node/week-1/lesson.md)
-  * [Week 17 - Middleware, Templating and APIs](/node/week-2/lesson.md)
-  * [Week 18 - Advanced Node](/node/week-3/lesson.md)
-* [Databases](/db/index.md)
-  * [Week 19 - Introduction to SQL](/db/week-1/lesson.md)
-  * [Week 20 - More SQL and integration with NodeJS](/db/week-2/lesson.md)
-  * [Week 21 - More integration with NodeJS](/db/week-3/lesson.md)
+- [Introduction](/introduction/index.md)
+  - [Week 1](/introduction/week-1/lesson.md)
```

Comment:
Does this changed intentionally to week1 instead of week 0  if so is it going to be tought  in the same week with `Semantic HTML and CSS`?